pyartifactory/models/repository.py

```diff
@@ -141,6 +141,7 @@ class SimpleRepository(BaseModel):
     description: Optional[str] = None
     url: str
     packageType: str
+    environments: List[str] = ['DEV']

```

Comment:
No need to add the environment here. The Get repositories result does not contain environment (See [Artifactory API](https://jfrog.com/help/r/jfrog-rest-apis/get-repositories))